packages/cicero-cli/test/cli.js

```diff
@@ -1269,3 +1292,43 @@ describe('#get', async () => {
     });
 });
 
+describe('#validateVerfiyArgs', () => {
+    it('no args specified', () => {
+        process.chdir(path.resolve(__dirname, 'data/signedArchive/'));
+        const args  = Commands.validateVerifyArgs({
+            _: ['verify']
+        });
+        args.template.should.match(/cicero-cli[/\\]test[/\\]data[/\\]signedArchive$/);
+    });
+    it('template arg specified', () => {
+        process.chdir(path.resolve(__dirname));
+        const args  = Commands.validateVerifyArgs({
+            _: ['verify', 'data/signedArchive/']
+        });
+        args.template.should.match(/cicero-cli[/\\]test[/\\]data[/\\]signedArchive$/);
+    });
+    it('verbose flag specified', () => {
```

Comment:
What are you testing in this test? Should there be an assertion?

Reply:
I noticed that for commands which had verbose passed flag to them, had a [similar test](https://github.com/accordproject/cicero/blob/2379c914031fa212d56a6171bdffcfc7d937b57b/packages/cicero-cli/test/cli.js#L131) conducted for them, so I did the same here.

Reply:
Ok, that's fair. Happy to let this one go.

Really, we should come back to all of these another time, as is this test doesn't prove anything (other than it doesn't throw?)